src/WaveTrack.cpp

```diff
@@ -145,7 +159,7 @@ WaveTrack::WaveTrack( const SampleBlockFactoryPtr &pFactory,
    mOldGain[0] = 0.0;
    mOldGain[1] = 0.0;
    mWaveColorIndex = 0;
-   SetDefaultName(TracksPrefs::GetDefaultAudioTrackNamePreference());
+   SetDefaultName(GetDefaultAudioTrackNamePreference());
```

Comment:
I cannot understand what's this "default name" for, and why do we need to keep a default name copy in each instance of the Track. It brings dependencies that seem unrelated to the WaveTrack or any other *Track.

Reply:
TrackName::GetDefaultName() is a little weird, and I didn't put it there.  I think researching the code history may trace it back to accessibility fixes: see where GetDefaultName() is used, to tell whether the name is defaulted.  But the default name for audio tracks is not hard-coded, but changeable by the user in preferences.

I'm not fixing all the existing weirdness but just breaking dependency cycles.